Makefile

```diff
@@ -83,9 +84,11 @@ dev_manifest:
 	sed -e "s~%IMAGE_NAME%~${IMAGE_NAME}~g" -e "s~%IMAGE_TAG%~${RELEASE_VERSION}~g" manifests/overlays/dev/kustomization.yaml.template > manifests/overlays/dev/kustomization.yaml
 
 .PHONY: generate
-generate:
+generate: goimports
 	go generate ./pkg/... ./cmd/...
 	hack/update-codegen.sh
+	# Workaround for https://github.com/kubernetes/kubernetes/issues/129774
+	$(GOIMPORTS) -w pkg/apis/kubeflow/v2beta1/zz_generated.defaults.go
```

Comment:
This is the most big change.
Since code-generator v1.32, the internal implementations for `import` have been changed (`FormatOnly` feature) and are less accurate to improve the generation performance. Thus, unused modules are technically inserted at the head of the Go file (`import` directive) outside of k/k repository.

So, in the short term, if we face such incorrect import directives, we should manually reformat them.